import-export-cli/impl/exportAPIPolicy.go

```diff
@@ -0,0 +1,77 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package impl
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+)
+
+// ExportAPIPolicyFromEnv function is used with export api policy command
+func ExportAPIPolicyFromEnv(accessToken, exportEnvironment, apiPolicyName, apiPolicyVersion, apiPolicyDefFormat string) (*resty.Response, error) {
+	apiPolicyEndpoint := utils.GetPublisherEndpointOfEnv(exportEnvironment, utils.MainConfigFilePath)
+	// var query string
```

Comment:
Do we need this?

Reply:
Fixed by [6e82281](https://github.com/wso2/product-apim-tooling/pull/924/commits/6e82281c87af32ef11a658fdc70bc37bbf5af147)